### ISSUE TITLE
Windows Installation tutorial has wrong cuDNN version requirement, 7 required for latest version.

### ISSUE THREAD

**Issue by nikitaverbis** [tensorflow/tensorflow#16950]
**Inaccuracies in the documentation**.

In the installation documentation Tensorflow in Windows says that the system must be running a version of CUDA 9.0 (correctly) and cuDNN 6.0 (not correct) because on the Nvidia website we can download and install only:
- **cuDNN** v7.0.5 , for **CUDA** 8.0/9.0/9.1 ;
- **cuDNN** v7.0.4 , for **CUDA** 8.0/9.0 ;
- **cuDNN** v6.0 , for **CUDA** 8.0 .

In https://developers.googleblog.com/2018/01/announcing-tensorflow-15.html said that 

> If you are using GPU Acceleration on **Windows** or Linux, TensorFlow 1.5 now has CUDA 9 and **cuDNN 7** support built-in. 

In issue #16477 (https://github.com/tensorflow/tensorflow/issues/16477)  as @gunan  said 
> Windows also requires cuDNN 7, looks like we missed that.

In proof of his words let me give you output logs:

(tensorflow15) C:\Windows\system32>python
Python 3.5.4 |Anaconda, Inc.| (default, Nov  8 2017, 14:34:30) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
      import tensorflow as tf
Traceback (most recent call last):
  File "C:\Users\User\AppData\Local\conda\conda\envs\tensorflow15\lib\site-packages\tensorflow\python\platform\self_check.py", line 87, in preload_check
    ctypes.WinDLL(build_info.cudnn_dll_name)
  File "C:\Users\User\AppData\Local\conda\conda\envs\tensorflow15\lib\ctypes\__init__.py", line 351, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: [WinError 126] Не найден указанный модуль (The specified module was not found)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\User\AppData\Local\conda\conda\envs\tensorflow15\lib\site-packages\tensorflow\__init__.py", line 24, in <module>
    from tensorflow.python import *
  File "C:\Users\User\AppData\Local\conda\conda\envs\tensorflow15\lib\site-packages\tensorflow\python\__init__.py", line 49, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Users\User\AppData\Local\conda\conda\envs\tensorflow15\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 30, in <module>
    self_check.preload_check()
  File "C:\Users\User\AppData\Local\conda\conda\envs\tensorflow15\lib\site-packages\tensorflow\python\platform\self_check.py", line 97, in preload_check
    % (build_info.cudnn_dll_name, build_info.cudnn_version_number))
ImportError: Could not find **'cudnn64_7.dll'**. TensorFlow requires that this DLL be installed in a directory that is named in your %PATH% environment variable. Note that installing cuDNN is a separate step from installing CUDA, and this DLL is often found in a different directory from the CUDA DLLs. You may install the necessary DLL by downloading **cuDNN 7** from this URL: https://developer.nvidia.com/cudnn

In conclusion.

Need to replace:

> **cuDNN  v6.0**. For details, see NVIDIA's documentation. Note that cuDNN is typically installed in a different location from the other CUDA DLLs. Ensure that you add the directory where you installed the cuDNN DLL to your %PATH% environment variable.

to

> **cuDNN v7.0**. For details, see NVIDIA's documentation. Note that cuDNN is typically installed in a different location from the other CUDA DLLs. Ensure that you add the directory where you installed the cuDNN DLL to your %PATH% environment variable.

on the web page https://www.tensorflow.org/install/install_windows

**Comment by cy89** [comment]
This looks to me like it's been fixed in the repository. 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/docs_src/install/install_windows.md

Please reopen if I've misread things.

**Comment by cy89** [comment]
And @nikitaverbis, thanks for reporting this!
